adapters/pubmatic/pubmatic.go

```diff
@@ -549,9 +561,24 @@ func (a *PubmaticAdapter) MakeBids(internalRequest *openrtb.BidRequest, external
 	for _, sb := range bidResp.SeatBid {
 		for i := 0; i < len(sb.Bid); i++ {
 			bid := sb.Bid[i]
+			impVideo := &openrtb_ext.ExtBidPrebidVideo{}
+
+			if len(bid.Cat) > 1 {
+				bid.Cat = []string{bid.Cat[0]}
```

Comment:
Consider using `bid.Cat = bid.Cat[0:1]` to avoid instantiating a new array.

Reply:
Added changes for the same